framework/src/wifi_manager/wifi_manager.c

```diff
@@ -654,29 +666,82 @@ wifi_manager_result_e _start_dhcpd(void)
 		WIFIADD_ERR_RECORD(ERR_WIFIMGR_SOFTAP_DHCPD_FAIL);
 	}
 	return wret;
+#else
+	struct netif *cur_netif;
+	cur_netif = netif_find(WIFIMGR_SOFTAP_IFNAME);
+	if (cur_netif == NULL) {
+		ndbg("[WM] DHCP Server - no network interface for dhcpd start\n");
+		WIFIADD_ERR_RECORD(ERR_WIFIMGR_CONNECT_DHCPD_FAIL);
+		return wret;
+	}
+	netif_set_up(cur_netif);
+<<<<<<< HEAD
+	if (dhcps_start(cur_netif) == ERR_OK) {
```

Comment:
I think this is merge conflict ?

Reply:
Two commits are merged by mistake. This PR will be recreated. Thanks.